src/components/navigation/header/config/DrawerNodeItems.tsx

```diff
@@ -1177,10 +1177,10 @@ export const drawerNodeItems = ({
         {
           translationKey: 'header.userMenu.contactDetails',
           link: {
-            de: '/de/member/masterdata/addressedit',
-            en: '/de/member/masterdata/addressedit',
-            fr: '/fr/member/masterdata/addressedit',
-            it: '/it/member/masterdata/addressedit',
+            de: '/de/profile',
+            en: '/de/profile',
+            fr: '/fr/profile',
+            it: '/it/profile',
           },
           visibilitySettings: {
             userType: {
```

Comment:
We didn't re-platformed profile page for professional sellers. So this link change need to be only for privates.
So, we need to create 2 links with the same translationKey in one to have visibilitySettings: { userType: { private: true, professional: false ... and in second  visibilitySettings: { userType: { private: false, professional: ftrue

Reply:
Changes done and tested from my side, can you check again? @AdzicMilos